beetmoverscript/test/test_script.py

```diff
@@ -141,88 +141,6 @@ def fake_list(*args):
             await push_to_maven(context)
 
 
-@pytest.mark.asyncio
```

Comment:
I initially attempted to fix these tests up to the point I realized they are no longer needed since we no longer have any zip extraction in the declarative artifacts world. The only assertion that can we could do it so to check that `artifacts_to_beetmove` are correct, but that's slightly redundant since the values stay the same, they are just being reorganized per locale in larger dict. Hence deleting these tests.

Reply:
Nit: I think we can keep these tests while this use-case is still supported.